serverless/reference.json

```diff
@@ -40866,7 +40866,8 @@
         },
         {
           "type": "string",
-          "minimum": 1
+          "minimum": 1,
+          "pattern": "\\$\\{file\\(.+\\.(yml|yaml)\\)\\}"
```

Comment:
Are you sure you need to double escape it? I don't think you need to do that, have a look at our private org repo below
https://github.com/online-directory/swagger-docs/blob/70cac6487bbf14fef92c27bb6f31ece8db2e7111/modules/shared.yaml#L1407

![image](https://user-images.githubusercontent.com/31315654/177208114-7b8e3bba-1f11-4895-ae18-056fddb2d0a4.png)

Reply:
That's yaml, and we get the same result when it's parsed to json

Reply:
got it